bluemix/src/main/java/com/ibm/watson/SpeechToText.java

```diff
@@ -43,120 +39,157 @@ public String speechToText(byte[] incomingBinaryPOSTData) {
     */
         HttpURLConnection conn = null;
         String serviceURL = "";
-           
+
         try {
-	    if (incomingBinaryPOSTData.length == 0) return XML_DEC + "<results>" + STATUS_OK + "<transcript />" + "</results>";
+            if (incomingBinaryPOSTData.length == 0) {
+                return XML_DEC + "<results>" + STATUS_OK + "<transcript />" + "</results>";
```

Comment:
I know this is just a reformat, but seeing this raises a question:

Why are we OK with 0 length post?

Whatever the answer is to that could be communicated by extracting that condition to a well-named function, such as `postDataIsSane` or something like that.